File: 백준/Bronze/5575. 타임 카드/타임 카드.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <ctype.h>

int main()
{
	for(int i = 0; i < 3; i++)
    {
        int a,b,c,d,e,f;
        scanf("%d %d %d %d %d %d",&a,&b,&c,&d,&e,&f);

        int hour = d - a;
        int minute = e - b;
        int second = f - c;

        if(second < 0)
        {
            minute -= 1;
            second += 60;
        }

        if(minute < 0)
        {
            hour -= 1;
            minute += 60;
        }

        printf("%d %d %d \n",hour,minute,second);
    }

	return 0;
}

File: 백준/Bronze/5575. 타임 카드/타임 카드.py
for i in range(3):
    list_time = list(map(int,input().split()))
    hour = list_time[3] - list_time[0]
    minute = list_time[4] - list_time[1]
    second = list_time[5] - list_time[2]

    if second < 0:
        second += 60
        minute -= 1

    if minute < 0:
        minute += 60
        hour -= 1
    
    print(hour, minute, second)